mitiq/interface/mitiq_pennylane/conversions.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2021 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions to convert between Mitiq's internal circuit representation and
+Pennylane's circuit representation.
+"""
+
+from cirq import Circuit, Moment
+
+from mitiq.interface.mitiq_qiskit import from_qasm as cirq_from_qasm, to_qasm
+from mitiq.utils import _pop_measurements
+
+from pennylane.wires import Wires
+from pennylane.tape import QuantumTape
+from pennylane.operation import Expectation
+
+from pennylane import from_qasm as pennylane_from_qasm
```

Comment:
```suggestion
from pennylane import from_qasm as pennylane_from_qasm

from pennylane_qiskit.qiskit_device import QISKIT_OPERATION_MAP

SUPPORTED_PL = set(QISKIT_OPERATION_MAP.keys())
UNSUPPORTED = {"CRX", "CRY", "CRZ", "S", "T"}
SUPPORTED = SUPPORTED_PL - UNSUPPORTED
```
This is for decomposing unsupported operations in the tape.